Add stash-cli install steps that use Homebrew on macOS

auxesis opened the pull request.

auxesis committed:
Add stash-cli install steps that use Homebrew on macOS

bennetthardwick commented on README.md:
````diff
@@ -62,22 +62,50 @@ We now have a running application with Sequelize and Express that we can use to
+brew install cipherstash/tap/stash
+```
+
+{% note %}
+You will need to grant an exception in System Settings the first time you run the binary.
+
+We will release a fix for this in Q2 2023.
+{% endnote %}
````
These don't seem to render how you'd expect.

```suggestion
Note: You will need to grant an exception in System Settings the first time you run the binary. We will release a fix for this in Q2 2023.
```

auxesis replied:
Ah, apparently you have to [also strong the text](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags), I'll fix that now.

auxesis replied:
Apparently GitHub currently implements two different ways of defining callouts in Markdown, one that [works in GitHub's docs](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags), and another that [works everywhere else Markdown is rendered on GitHub](https://github.com/orgs/community/discussions/16925). 

Have fixed the syntax in 7512a33e228173dd011657627092958716776317, looks like this now:

<img width="858" alt="image" src="https://github.com/cipherstash/cipherstash-sequelize-demo-app/assets/12306/7de71bc3-5c81-4a41-ab51-da0d80428670">

bennetthardwick reviewed (approved):
Lgtm, just that `{% note %}` block doesn't work

auxesis committed:
Try and make the note render properly

Per https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags

auxesis committed:
Try different syntax to make the note render correctly

Per https://github.com/orgs/community/discussions/16925
